Automattic/wp-calypso
# Listing content on /advertising by number of views - POC 1

Related to 1693-gh-Automattic/dotcom-forge

## Proposed Changes

* Here we're showing /advertising posts/pages by: 
  - 5 most recent posts
  - up to 20 most viewed
  - up to 20 most commented (if doesn't find 10 most viewed)
  - up to 20 products

## Testing Instructions



* Apply the Diff in your sandbox: https://github.com/Automattic/jetpack/pull/29149
* Access a popular site you have and make sure we're listing posts and pages by their views at the top of /advertising items
* Tip: Comment [these lines](https://github.com/Automattic/wp-calypso/blob/update/filter-advertising-by-views-v2/client/my-sites/promote-post/main.tsx#L123-L144) and access any private site from automattic that have more views, like the explorers p2.
* Test around with Atomic (will need to sync Jetpack repo to a WoA blog) and Simple sites
* Make sure it works with sites without any views (in this case we'll load pages and posts by comments):

<img width="867" alt="image" src="https://user-images.githubusercontent.com/1044309/221255385-377cdcbe-c2be-4a9e-867d-b8653c976c99.png">


## Pre-merge Checklist
